### ISSUE TITLE
OpenShift support

### ISSUE THREAD

**Issue by akostadinov** [RedHatQE/webdriver-wharf#14]
Allow launching containers in a remote OpenShift cluster. Probably images tuning would be needed because in OpenShift containers run unprivileged by default.

**Comment by jchristi** [comment]
Related to #6

**Comment by akostadinov** [comment]
No, nothing related really. The only common is `remote` but that ends it.

**Comment by jchristi** [comment]
If webdriver-wharf doesn't support anything remote, then it doesn't support OpenShift. If webdriver-wharf is changed to not assume all containers are local, then it _can_ support OpenShift.

**Comment by akostadinov** [comment]
yep, container interaction would need to be abstracted out
